addons/frontend/src/svg-doc.ts

```diff
@@ -51,6 +55,10 @@ export class SvgDocument {
     /// for ctrl-wheel rescaling
     this.hookedElem.style.transformOrigin = "0px 0px";
 
+    /// Style fields
+    this.borderColor = getComputedStyle(document.documentElement).getPropertyValue('--vscode-editor-background')
+      || 'rgba(0, 0, 0, 0.5)';
```

Comment:
how does rgba 000 0.5 looks like?

Reply:
I guess your would love to set fallback color to `rgb(82, 86, 89)` (color name: `trout`), which is used by many PDF viewers.